scss/civicrm/cases/_search.scss

```diff
@@ -0,0 +1,76 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+#{civi-page('case-search')} {
+  .crm-accordion-body .form-layout {
+    display: block;
+    margin-top: $crm-standard-gap;
+
+    > tbody:first-child {
+      display: block;
+
+      > tr {
+        display: block;
+
+        > td {
+          display: block;
+          margin-left: $crm-standard-gap;
+
+          // Display labels as blocks for margins and paddings consistency
+          > label:first-child {
+            display: block;
+
+            // Remove <br>s after labels because labels are displayed as blocks
+            + br {
+              display: none;
+            }
+          }
+        }
+
+        &:nth-of-type(1) {
+          > td {
+            // Brings "Client Name or Email" and the input to individual lines
+            &:nth-of-type(1) {
+              label {
+                display: block;
+
+                + input {
+                  margin-left: -0.6em;
```

Comment:
why `0.6em`? do we have a var for this?

Reply:
We ar discarding `&nbsp;` here that are hardcoded by Civi. I can leave a comment in SCSS. Would that help?

Reply:
yes please.